tests/parm/control_gdas_wav.nml.IN

```diff
@@ -0,0 +1,338 @@
+  phys_hydrostatic = .false.
+  use_hydro_pressure = .false.
+  beta = 0.
+  a_imp = 1.
+  p_fac = 0.1
+  k_split = 2
+  n_split = 6
+  nwat = 6
+  na_init = @[NA_INIT]
+  d_ext = 0.
+  dnats = 1
+  fv_sg_adj = 450
+  d2_bg = 0.
+  nord = 2
+  dddmp = 0.1
+  d4_bg = 0.12
+  vtdm4 = 0.02
+  delt_max = 0.002
+  ke_bg = 0.
+  do_vort_damp = .true.
+  external_ic = @[EXTERNAL_IC]
+  external_eta = .true.
+  gfs_phil = .false.
+  nggps_ic = @[NGGPS_IC]
+  mountain = @[MOUNTAIN]
+  ncep_ic = .false.
+  d_con = 1.
+  hord_mt = 5
+  hord_vt = 5
+  hord_tm = 5
+  hord_dp = -5
+  hord_tr = 8
+  adjust_dry_mass = .true.
+  dry_mass=98320.0
+  consv_te = 1.
+  do_sat_adj = .true.
+  consv_am = .false.
+  fill = .true.
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = .false.
+  res_latlon_dynamics = ""
+  
+/
+
+&cires_ugwp_nml
+       knob_ugwp_solver  = 2
+       knob_ugwp_source  = 1,1,0,0
+       knob_ugwp_wvspec  = 1,25,25,25
+       knob_ugwp_azdir   = 2,4,4,4
+       knob_ugwp_stoch   = 0,0,0,0
+       knob_ugwp_effac   = 1,1,1,1
+       knob_ugwp_doaxyz  = 1
+       knob_ugwp_doheat  = 1
+       knob_ugwp_dokdis  = 1
+       knob_ugwp_ndx4lh  = 1
+       knob_ugwp_version = 0
+       launch_level      = 54                   
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = @[NPZP]
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&gfs_physics_nml
+  fhzero       = 6
+  h2o_phys     = .true.
+  ldiag3d      = .false.
+  fhcyc        = 1
+  use_ufo      = .true.
+  pre_rad      = .false.
+  ncld         = 5
+  imp_physics  = 11
+  pdfcld       = .false.
+  fhswr        = 3600.
+  fhlwr        = 3600.
+  ialb         = 1
+  iems         = 1
+  iaer         = 5111
+  icliq_sw     = 2
+  iovr         = 3
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = .true.
+  shal_cnv     = .true.
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = .false.
+  satmedmf     = .true.
+  isatmedmf    = 1
+  lheatstrg    = .false.
+  random_clds  = .false.
+  trans_trac   = .true.
+  cnvcld       = .true.
+  imfshalcnv   = 2
+  imfdeepcnv   = 2
+  cdmbgwd      = 1.1,0.72,1.0,1.0
+  prslrd0      = 0.
+  ivegsrc      = 1
+  isot         = 1
+  lsoil        = 4
+  lsm          = 1
+  iopt_dveg    = 1
+  iopt_crs     = 1
+  iopt_btr     = 1
+  iopt_run     = 1
+  iopt_sfc     = 1
+  iopt_frz     = 1
+  iopt_inf     = 1
+  iopt_rad     = 1
+  iopt_alb     = 2
+  iopt_snf     = 4
+  iopt_tbot    = 2
+  iopt_stc     = 1
+  debug        = .false.
+  oz_phys      = .F.
+  oz_phys_2015 = .T.
+  nstf_name    = 2,0,0,0,0
+  nst_anl      = .true.
+  psautco      = 0.0008,0.0005
+  prautco      = 0.00015,0.00015
+  lgfdlmprad   = .true.
+  effr_in      = .true.
+  cplwav       = .true.
```

Comment:
Instead of adding an entire new file, can cplwav just be made a variable input?  I think that helps have fewer files to change as we move forward.

Reply:
I agree w/ Jessica here. If the only difference between ``control_gdas.nml.IN`` and ``control_gdas_wav.nml.IN`` is the flip of the ``cplwav`` to true, you should use the existing ``control_gdas.nml.IN`` by introducing the ``[@CPLWAV ]`` variable.

Reply:
Done